packages/exos-scripts/package.json

```diff
@@ -34,6 +34,8 @@
     "copy-webpack-plugin": "^5.1.1",
     "cross-spawn": "^7.0.2",
     "css-loader": "^3.4.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
```

Comment:
Add `react` and `react-dom` as peer dependencies, with its corresponding version that matches enzyme.